CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
  - #2255, Apply `to_tsvector()` explicitly to the full-text search filtered column (excluding `tsvector` types) - @laurenceisla
  - #1578, Log the main SQL query to stderr at the current `log-level` when `log-query=main-query` - @laurenceisla
  - #3903, Log connection pool borrows on `log-level=debug` - @taimoorzaeem
+ - #3906, Add `PGRST125` and `PGRST126` errors - @taimoorzaeem
```

Comment:
I think this should be a fix, given the description of https://github.com/PostgREST/postgrest/issues/3906
```suggestion
```